helm/hookshot/values.yaml

```diff
@@ -0,0 +1,171 @@
+  tag:
+
+# -- List of names of k8s secrets to be used as ImagePullSecrets for the pod
+imagePullSecrets: []
+
+# -- Name override for helm chart
+nameOverride: ""
+
+# -- Full name override for helm chart
+fullnameOverride: ""
+
+serviceAccount:
+  # -- Specifies whether a service account should be created
+  create: true
+  # -- Annotations to add to the service account
+  annotations: {}
+  # -- The name of the service account to use. If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+# -- Extra annotations for Hookshot pod
+podAnnotations: {}
+
+# -- Pod security context settings
+podSecurityContext: {}
+# fsGroup: 2000
+
+# -- Security context settings
+securityContext: {}
+# capabilities:
+#   drop:
+#   - ALL
+# readOnlyRootFilesystem: true
+# runAsNonRoot: true
+# runAsUser: 1000
+
+
+service:
+  # -- Service type for Hookshot service
+  type: ClusterIP
+  # -- Port for Hookshot service
+  port: 80
+  # -- Extra annotations for service
+  annotations: {}
+  # -- Extra labels for service
+  labels: {}
+
+  webhook:
+    # -- Webhook port as configured in container
+    port: 9000
+  metrics:
+    # -- Metrics port as configured in container
+    port: 9001
+  appservice:
+    # -- Appservice port as configured in container
+    port: 9002
+
+ingress:
+  webhook:
+    # -- Enable ingress for webhook
+    enabled: false
+    # -- Ingress class name for webhook ingress
+    className: ""
+    # -- Annotations for webhook ingress
+    annotations: {}
+    # -- Host configuration for webhook ingress
+    hosts: []
+    # -- TLS configuration for webhook ingress
+    tls: []
+
+  appservice:
+    # -- Enable ingress for appservice
+    enabled: false
+    # -- Ingress class name for appservice ingress
+    className: ""
+    # -- Annotations for appservice ingress
+    annotations: {}
+    # -- Host configuration for appservice ingress
+    hosts: []
+    # -- TLS configuration for appservice ingress
+    tls: []
+
+# -- Pod resource requests / limits
+resources: {}
+# We usually recommend not to specify default resources and to leave this as a conscious
+# choice for the user. This also increases chances charts run on environments with little
+# resources, such as Minikube. If you do want to specify resources, uncomment the following
+# lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+# limits:
+#   cpu: 100m
+#   memory: 128Mi
+# requests:
+#   cpu: 100m
+#   memory: 128Mi
+
+autoscaling:
+  enabled: false
+
+# -- Node selector parameters
+nodeSelector: {}
+
+# -- Tolerations for deployment
+tolerations: []
+
+# -- Affinity settings for deployment
+affinity: {}
+
+hookshot:
+  # -- Name of existing ConfigMap with valid Hookshot configuration
+  existingConfigMap:
+
+  # -- Raw Hookshot configuration. Gets templated into a YAML file and then loaded unless an existingConfigMap is specified.
+  config:
```

Comment:
Wondering if there is a way to reference our existing sample config, lest this gets out of date.

Reply:
I was thinking about this too - 

A quick hack would be to have a little script along side the chart that pulls the latest example config and dumps it into the values before building the chart and have that pre-execute in the CI pipeline

Reply:
I think that would be good, if we don't do this it's just going to get buggy and out of date.

Reply:
How often does the config structure change? Are there higher level keys that remain consistent - i.e. could our default values.yaml config be a stub that won't change?

Reply:
Anyways, I sorted this out in latest commit - Will use some `yq` magic to dump the current sample config in as default values on each chart release.